.github/workflows/run_tests.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
     - name: Install
       run: |
-        pip install .
+        python setup.py install
```

Comment:
why did you change this? Pip is much better as package manager than the setuputils. I would honestly think the better way to install.

Reply:
Oh, that was to make sure the `setup.py` script actually worked. I'll change it back.

Reply:
```suggestion
        pip install .
```